packages/angular/cli/models/command.ts

```diff
@@ -173,7 +173,6 @@ export abstract class Command<T extends BaseCommandOptions = BaseCommandOptions>
       return this.printJsonHelp(options);
     } else {
       const startTime = +new Date();
-      await this.reportAnalytics([this.description.name], options);
```

Comment:
With this change, I think non-architect and non-schematic commands will also need to manually run `reportAnalytics`.

Reply:
You are right!

Reply:
PTAL